agent/wptdriver/wpt_driver_core.cc

```diff
@@ -351,12 +339,18 @@ bool WptDriverCore::BrowserTest(WptTestDriver& test, WebBrowser &browser) {
   WptTrace(loglevel::kFunction, 
             _T("[wptdriver] WptDriverCore::BrowserTest done\n"));
 
+  // activate logs

```

Comment:
Why did we move it here? This would cause the first run to not log anywhere. Right?